plugins/woocommerce/tests/php/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -23,7 +23,7 @@ public function print_custom_product_type_add_to_cart_markup() {
 	 * Tests that the correct content is rendered for each product type.
 	 */
 	public function test_product_type_add_to_cart_render() {
-		add_filter( 'woocommerce_custom_add_to_cart', array( $this, 'print_custom_product_type_add_to_cart_markup' ) );
+		add_action( 'woocommerce_custom_add_to_cart', array( $this, 'print_custom_product_type_add_to_cart_markup' ) );
```

Comment:
Indeed this seems to be an action, but I think there's another test using this wrongly in `/plugins/woocommerce/tests/php/src/Blocks/BlockTypes/AddToCartWithOptions.php`, we could fix that one also. Not sure why it's not failing though 🤔

Reply:
Oh no! I had auto-merge enabled.

But I think it should be fine, this change was just merely a convention. In general, WP won't care a lot if you use actions or filters, it kinda equalizes them in some cases :)